tests/e2e/specs/blocks/shopper/shopper-wc-blocks-checkout-failures.spec.js

```diff
@@ -22,7 +22,7 @@ import {
 	fillCardDetailsWCB,
 	clearWCBCardDetails,
 	confirmCardAuthentication,
-} from '../../utils/payments';
+} from '../../../utils/payments';
```

Comment:
If we add:
```js
export * from './payments';
```
in the `tests/e2e/utils/index.js`, we wouldn't have to make this change in 2 places and can also remove separate import for payments. Can be done separately in a different PR also, just suggesting it here.

Reply:
@deepakpathania Let's do the optimization on a different PR